client/src/app/App.js

```diff
@@ -1501,6 +1501,18 @@ export class App extends PureComponent {
       return this.logEntry(message, category, action);
     }
 
+    if (action === 'deploy-diagram') {
```

Comment:
Question to self / you: What would it take to make `deploy` an actual plug-in?

* the app does not know anything about it
* it can be added and removed as wished

Could we already build it in such way?

Reply:
Not sure if this is easily doable at the moment, as we need to communicate with the backend from the client with custom actions.